feature-libs/product-configurator/rulebased/components/overview-filter-button/configurator-overview-filter-button.component.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { ConfiguratorRouterExtractorService } from '@spartacus/product-configurator/common';
+import { LaunchDialogService, LAUNCH_CALLER } from '@spartacus/storefront';
+import { Observable, OperatorFunction } from 'rxjs';
+import { filter, switchMap } from 'rxjs/operators';
+import { ConfiguratorCommonsService } from '../../core/facade/configurator-commons.service';
+import { Configurator } from '../../core/model/configurator.model';
+
+@Component({
+  selector: 'cx-configurator-overview-filter-button',
+  templateUrl: './configurator-overview-filter-button.component.html',
+})
+export class ConfiguratorOverviewFilterButtonComponent {
+  constructor(
+    protected launchDialogService: LaunchDialogService,
+    protected configuratorCommonsService: ConfiguratorCommonsService,
+    protected configRouterExtractorService: ConfiguratorRouterExtractorService
+  ) {}
+
+  @ViewChild('filterButton') filerButton: ElementRef;
```

Comment:
@Uli-Tiger Please rename
```suggestion
  @ViewChild('filterButton') filterButton: ElementRef;
```